nf_core/pipeline-template/.pre-commit-config.yaml

```diff
@@ -1,4 +1,10 @@
 repos:
+  - repo: https://github.com/astral-sh/ruff-pre-commit
```

Comment:
I'm not sure if we should be as strict with the python code in pipelines and force all the changes that we added to tools with ruff

Reply:
yes, me neither, but I was thinking because we already have it in the pyproject.toml we should have it also in pre-commit. or should was your thought with having it in the pyproject, that people can use it if they want, but don't have to?

Reply:
I probably didn't think about it when I added it to the pyproject.toml, but maybe it's good to have it as an option with the same configuration as in tools, but not mandatory.